src/dc_tools.rs

```diff
@@ -50,24 +46,19 @@ pub unsafe fn dc_strdup_keep_null(s: *const libc::c_char) -> *mut libc::c_char {
 }
 
 pub unsafe fn dc_atoi_null_is_0(s: *const libc::c_char) -> libc::c_int {
-    return if !s.is_null() { atoi(s) } else { 0i32 };
+    if !s.is_null() {
+        to_str(s).parse().unwrap_or_default()
```

Comment:
Does `.parse()` automatically convert to int here?

Reply:
that is exactly what `parse` does